modules/ROOT/pages/obtain-connected-apps-credentials.adoc

```diff
@@ -0,0 +1,27 @@
+= Obtain Connected Apps Credentials
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, Connected Apps credentials
+
+Connected Apps enables you to seamlessly integrate external applications with Anypoint Platform. Using Connected Apps, you can provide access to external applications without having to share your user credentials. 
+
+To create a Connected App and obtains its credentials:
+
+. From Anypoint Platform, select *Access Management* > *Connected Apps* > *Create App*.
+. In the *Name* field, specify a name for the app.
+. In the *Type* field, select the *App acts on its own behalf (client credentials)* option.
+. Click *Add Scopes* and select the *Exchange Contributor* and *Manage APIs Configuration* scopes.
++
+* For the `Exchange contributor` scope, you must select the required business groups.
+* For the `Manage APIs Configuration` scope, you must select the required business groups and environments,
+. Click *Next* and select the business group and environments to add the scope.
++
+The *Review* button is greyed out until you choose an environment for each of the added business groups.
+. Click *Review* and then click *Add Scopes*.
+. Click *Save*.
```

Comment:
Please review lines 14-21 for accuracy of chronology; it looks to me like actions are repeated and I want to be sure you intended that.

Reply:
Done! Thanks!